src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.tsx

```diff
@@ -167,7 +168,7 @@ const QDMReporting = () => {
                 Rate Aggregation
               </FieldLabel>
```

Comment:
I think `FieldLabel` would be gone as well. Can use `label` prop of `TextArea` `label="Rate Aggregation"`